src/main/java/seedu/address/logic/Messages.java

```diff
@@ -42,7 +42,6 @@ public static String format(Person person) {
                 .append("; Email: ")
                 .append(person.getEmail())
                 .append("; Address: ")
-                .append(person.getAddress())
```

Comment:
What was the purpose of deleting just this line? Why not delete "Address:" as well?

Reply:
Deleted! Thanks!